### ISSUE TITLE
Cannot use Apex to backward the loss generated by SegFormer

### ISSUE THREAD

**Issue by QIU023** [NVlabs/SegFormer#56]
I had used the SegFormer and MixTransformer you provided to switch the deeplab model in my experiments. Hence the model and optimizer need to be amp.initialized before training. However, when the logit generated by the model will lead to the amp.scale_loss(loss, optim) to raise the problem: CUDNN_STATUS_NOT_SUPPORTED when loss.backward() is executed, where the loss refers to CrossEntropyLoss.
My hyperparameters are batchsize=24, resize=512*512, O1 optimize of apex, which is a normal size in Semantic Segmentation.
Could you please help me that the Segformer training using apex to accelerate, as without apex the experiment of voc will not be fast enough even if I have V100 GPU. 
Thanks for your help!
